registration/register.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
+  <link rel="stylesheet" type="text/css" href="frontend/main.css">
+  <title>Registration For SPECTRA</title>
```

Comment:
Can we please reorder the head?
https://github.com/SpectraHackathon/spectra-revamp/blob/master/team/teampage.html

The links for css and js are usually done last.